managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorContainer.java

```diff
@@ -164,6 +175,22 @@ public ManagedCursor getSlowestReader() {
         }
     }
 
+    public boolean hasCursors() {
```

Comment:
It's becoming a bit confusing now to understand the meaning of `hasCursors()` and `isEmpty()`. We should be use a more unambiguous/explicit naming here.

My suggestion: 
 * `hasCursors()` --> `isEmtpy()` // Since we're talking about a cursors container
 * `isEmpty()` --> `hasDurableCursors()`

(and fix the reversal of the boolean logic)